app/src/test/java/com/xmartlabs/template/helper/GsonRequiredTest.java

```diff
@@ -0,0 +1,117 @@
+package com.xmartlabs.template.helper;
+
+import com.google.gson.ExclusionStrategy;
+import com.google.gson.FieldAttributes;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.xmartlabs.template.service.gson.GsonExclude;
+import com.xmartlabs.template.service.gson.GsonRequired;
+import com.xmartlabs.template.service.gson.JsonRequiredFieldException;
+import com.xmartlabs.template.service.gson.RequiredFieldDeserializer;
+
+import junit.framework.Assert;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import lombok.Builder;
+import lombok.Data;
+
+public class GsonRequiredTest {
```

Comment:
Damn, I missed it!

btw, I think these 2 lines can go away as the 2.2 version of the Android Gradle plugin automatically installs them: https://github.com/xmartlabs/Android-Base-Project/blob/master/circle.yml#L5-L6